Model/Transaction.php

```diff
@@ -106,6 +106,23 @@ public function getChildTransactions()
         return $this->_aChildTransactions;
     }
 
+    /**
+     * Returns true is transaction's payment method is Payment on Invoice or Payment in Advance
+     *
+     * @return bool
+     *
+     * @since 1.3.0
+     */
+    public function isPoiPiaPaymentMethod()
+    {
+        $sPaymentId = $this->getPaymentType();
+        if ($sPaymentId === 'wdpaymentoninvoice' || $sPaymentId === 'wdpaymentinadvance') {
```

Comment:
I think it would be better to use `PaymentOnInvoicePaymentMethod::getName()` for this, or even better, check if the payment is a subclass of `BasePoiPiaPaymentMethod`, e.g.:
```php
$sPaymentId = $this->getPaymentType();
$oPayment = PaymentMethodHelper::getPaymentById($sPaymentId);

return is_subclass_of($oPayment->getPaymentMethod(), BasePoiPiaPaymentMethod::class);
```

Reply:
Nice 👍